services/fetcher/randomQuoteFetcher.js

```diff
@@ -0,0 +1,31 @@
+const axios = require("axios");
+const fetchAnimeImage = require("./animeImageFetcher");
+const fetchCharacterImage = require("./characterImageFetcher");
+
+const config = {
+  method: "GET",
+  url: "http://animechan.melosh.space/random",
+};
+
+const fetchRandomQuote = async () => {
+  try {
+    const response = await axios.request(config);
+
+    // fetching anime image
+    let animeImageURI = await fetchAnimeImage(response.data.anime || "");
+
+    // fetching character image
+    let characterImageURI = await fetchCharacterImage(
+      response.data.character || ""
+    );
+
+    const quote = { ...response.data, animeImageURI, characterImageURI };
```

Comment:
`const quote = { ...response.data, animeImageURI: animeImageURI, characterImageURI: characterImageURI };`

won't it be like this?

Reply:
if we don't pass key-value pair then the name of the variable automatically becomes the key,